src/main/kotlin/net/irisshaders/lilybot/extensions/moderation/TemporaryModeration.kt

```diff
@@ -113,40 +115,83 @@ class TemporaryModeration : Extension() {
 						}
 						return@action
-					// If the moderator ping role is in roles, return
+						// If the moderator ping role is in roles, return
 					} else if (moderatorRoleId in roles) {
 						respond {
 							content = "You cannot warn moderators!"
 						}
 						return@action
 					}
-				// Just to catch any errors in the checks
+					// Just to catch any errors in the checks
 				} catch (exception: Exception) {
 					logger.warn("IsBot and Moderator checks skipped on `Warn` due to error")
 				}
 
-				val oldPoints = DatabaseHelper.getWarn(userArg.id, guild!!.id)
-				val newPoints = oldPoints.plus(arguments.warnPoints)
-				DatabaseHelper.setWarn(userArg.id, guild!!.id, newPoints)
-				val databasePoints = DatabaseHelper.getWarn(userArg.id, guild!!.id)
+				val oldStrikes = getWarn(userArg.id, guild!!.id)
+				DatabaseHelper.setWarn(userArg.id, guild!!.id, oldStrikes.plus(1))
+				val newStrikes = getWarn(userArg.id, guild!!.id)
 
 				// DM the user about the warning
 				val dm = userDMEmbed(
 					userArg,
 					"You have been warned in ${guild?.fetchGuild()?.name}",
-					"You were given ${arguments.warnPoints} points\n" +
-							"Your total is now $databasePoints\n\n**Reason:**\n${arguments.reason}",
+					"You were given a strike\n" +
+							"Your total is now $newStrikes\n\n**Reason:**\n${arguments.reason}\n\nFor more information" +
+							" about the warn system, please see [this document](https://github.com/IrisShaders/LilyBot" +
+							"/blob/develop/docs/warn-strikes.md)",
 					null
 				)
 
-				// Check the points amount, before running sanctions
-				if (databasePoints in (75..124)) {
+				respond {
+					content = "Warned User"
+				}
+
+				actionLog.createEmbed {
+					title = "Warning"
+					color = DISCORD_BLACK
+					timestamp = Clock.System.now()
+
+					field {
+						name = "User:"
+						value = "${userArg.tag} \n${userArg.id}"
+						inline = false
+					}
+					field {
+						name = "Total Strikes:"
+						value = newStrikes.toString()
+						inline = false
+					}
+					field {
+						name = "Reason:"
+						value = arguments.reason
+						inline = false
+					}
+					field {
+						name = "User notification"
+						value =
+							if (dm != null) {
+								"User notified with a direct message"
+							} else {
+								"Failed to notify user with a direct message"
+							}
+						inline = false
+					}
+					footer {
+						text = "Requested by ${user.asUser().tag}"
+						icon = user.asUser().avatar?.url
+					}
+				}
+
+
+				// Check the amount of points before running sanctions
+				if (newStrikes == 2) {
 					userDMEmbed(
 						userArg,
-						"You have been timed-out in ${guild!!.fetchGuild().name}",
-						"You have accumulated too many warn points, and have hence been given a " +
-								"3 hour timeout",
-						DISCORD_BLACK
+						"This is your second warning in ${guild?.fetchGuild()?.name}",
+						"You have been timed out for 3 hours. Further warnings will result in a 12 hour " +
```

Comment:
Not all further warnings will be 12 hours, maybe say "Your next warning" instead. This line is also too long.